pkg/ccl/backupccl/backup_processor.go

```diff
@@ -626,9 +628,8 @@ func (s *sstSink) Close() error {
 		s.cancel()
 	}
 
-	// Release the memory reserved for the file buffer back to the memory
-	// accumulator.
-	s.backupMem.release(smallFileBuffer.Get(s.conf.settings))
+	// Release the memory reserved for the file buffer.
+	s.memAcc.Shrink(s.ctx, smallFileBuffer.Get(s.conf.settings))
```

Comment:
I know this is in the existing code, but this should really read the the setting once before it goes to Grow and then store how much it asked for and then release _that_ -- Shrinking the current value of the setting does not really ensure we shrieked by as much as we grew.

Reply:
good point, done.